service/src/test/java/uk/nhs/adaptors/gp2gp/common/configuration/CustomTrustStoreTest.java

```diff
@@ -48,6 +47,11 @@ static void setUp() {
                            software.amazon.awssdk.core.sync.RequestBody.fromFile(trustStoreFile));
     }
 
+    @BeforeEach
+    void prepare() {
+        customTrustStore = new CustomTrustStore(s3Client);
```

Comment:
Minor, why not put it in the existing `@BeforeAll` ?

Reply:
yes, I can do